dataland-inbound-proxy/docker-entrypoint.d/03-configure_robots_file.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+datalandServerName="dataland.com"
+filePath="/var/www/html/robots.txt"
+delimiter="/"
+
+set -exo pipefail
+echo "Letsencrypt path:"$PROXY_LETSENCRYPT_PATH
+fullServerPath=$PROXY_LETSENCRYPT_PATH
+
+serverName="${fullServerPath##*"$delimiter"}"
+echo serverName
+set -u
+
+devText="User-agent: *\nDisallow: /"
+datalandText="User-agent: *\nAllow: / \nDisallow: /keycloak/"
+
+scriptDir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
This will provide the correct path, but seems a little overcomplicated.

Why not like this?:

`scriptDir="$( dirname "${BASH_SOURCE[0]}" )"`

Reply:
done